volumes/ecocounter/readme.md

```diff
@@ -120,13 +121,23 @@ The `ecocounter_pull` DAG runs daily at 3am to populate `ecocounter` schema with
 
 - `update_sites_and_flows` task identifies any sites and "flows" (known as channels in the API) in the API which do not exist in our database and adds them to `ecocounter.sites_unfiltered` and `ecocounter.flows_unfiltered`. The new rows contain a flag `validated = null` indicating they still need to be manually validated. A notification is sent with any new additions.  
 - `pull_ecocounter` task pulls data from the Ecocounter API and inserts into the `ecocounter.counts_unfiltered` table. 
-
+- `done` is an external task marker to trigger the `ecocounter_check` DAG for additional "yellow card" data checks.  
+   
 ### `data_checks` TaskGroup
 This task group runs data quality checks on the pipeline output.  
-- `check_volume` checks the sum of volume in `ecocounter.counts` view and notifies if less than 70% of the 60 day lookback avg.  
-- `check_distinct_flow_ids` checks the count of distinct flow_ids appearing in `ecocounter.counts` view and notifies if less than 70% of the 60 day lookback avg.  
+- `check_volume` checks the sum of volume in `ecocounter.counts` (filtered view) and notifies if less than 70% of the 60 day lookback avg.  
+- `check_distinct_flow_ids` checks the count of distinct flow_ids appearing in `ecocounter.counts` (filtered view) and notifies if less than 70% of the 60 day lookback avg.  
```

Comment:
I feel like this might not catch / tell us much.  It would need to be widespread outage to affect decreased flow_id across the board. 🤔 And in case of widespread outage maybe site_id is a easier to understand identifier? 

In term of additional data checks : 
I do think some checks on `flow_id` would be very helpful. Maybe something flow_id + site_id? I dont work with this data enough to know if this combination check is too sensitive. 
Or just individual volume checks on distinct site_id's volume 🤔 Similar to the large variation alerts we get from ecocounter alert emeow

Reply:
That alert did go off recently: https://bditto.slack.com/archives/C021YMH9BK5/p1714719679151429
Agreed it's not particularly useful, especially with the `check_site_outages` task which has some overlap... 

The `check_site_outages` task (SQL here: `volumes/ecocounter/data_checks/select-ongoing_outages.sql`) does check for both downed sites & flows! Specifically see the `GROUPING SETS` + 2nd CTE. However there are no current examples were one specific flow isn't working but others at the site are.